admin/qtx_configuration.php

```diff
@@ -1392,7 +1392,7 @@ function qtranxf_conf() {
 					</fieldset><br />
 					<textarea id="highlight_mode_custom_css" name="highlight_mode_custom_css" style="width:100%"><?php echo esc_attr($highlight_mode_custom_css); ?></textarea>
 					<br />
-					<small><?php echo __('To reset to default, clear the text.', 'qtranslate').' '; printf(__('The color in use is taken from your profile option %s, the third color.', 'qtranslate'), '"<a href="'.admin_url('/profile.php').'">'.__('Admin Color Scheme').'</a>"') ?></small>
+					<small><?php echo __('To reset to default, clear the text.', 'qtranslate').' '; printf(__('The color in use is taken from your profile option %s, the third color.', 'qtranslate'), '"<a href="'.admin_url('/profile.php').'">'.__('Admin Color Scheme', 'qtranslate').'</a>"') ?></small>
```

Comment:
This one was intentional. It is translated by WordPress, since it is a string from the main installation. It goes to .pot file anyway, it does not hurt to have it there translated, but if there is no .po file for a language, it will be translated anyway from WordPress, if domain is not specified. In this case, it is important to keep translation the same as in WP, since otherwise a person may not recognize, which option is meant.

I know, we discussed a few similar cases, and I am not sure every time. It seems to me that it is better to have at least a few strings translated in the absence of .po, than nothing. It does not really make a difference to translator, since it gets to .pot file in any case. What do you think?